dialogue-core/src/main/java/com/palantir/dialogue/core/ActiveRequestInstrumentationChannel.java

```diff
@@ -29,27 +28,22 @@
 
     private final String stage;
     private final Channel delegate;
-    private final DialogueClientMetrics metrics;
+    private final Counter active;
 
     ActiveRequestInstrumentationChannel(
-            Channel delegate, @CompileTimeConstant String stage, DialogueClientMetrics metrics) {
+            Channel delegate, String serviceName, @CompileTimeConstant String stage, DialogueClientMetrics metrics) {
```

Comment:
There's an argument for both the endpoint service name and target name depending on the use case. We could do both, and come back to remove the one that we find less helpful.